src/main/resources/avro/action.avsc

```diff
@@ -53,6 +53,25 @@
       },
       "default": []
     },
+    {
+      "name": "params",
+      "type": [
+        {
+          "type": "record",
+          "name": "PoliciesParams",
```

Comment:
@pilhuhn WDYT of this?
I'm thinking into having a specific field in the avro schema for each app. It would have specific parameters for the email templates as they require.

Reply:
I was thinking of having a map <prop-name, value> where the prop-name can be used in Qute templates and the value is then inserted into the template.
The template itself would be determined by application name + additional properties like 'email' or 'daily-email' or webhook or so.

Reply:
These were my initial thoughts. The problem is that policies uses a trigger array that can't be represented in that way (unless value is an array or map itself). 

Maybe that params is just a (json) string and we accept whatever the application wants to send?

That way we decode it as json and send the params to the template engine.

Reply:
> The template itself would be determined by application name + additional properties like 'email' or 'daily-email' or webhook or so.

+1
There is a field call "message" That I'm reserving for the webhook data (pass whatever is there). But we could have a webhook template too.